spec/features/employer_setup_spec.rb

```diff
@@ -193,7 +194,7 @@
   end
 end
 
-feature 'admins can approve an employer', js: true, driver: :webkit do
+feature 'admins can approve an employer', js: true, driver: :poltergeist do
```

Comment:
I think `js: true` along with the default driver specification in the rails helper should trigger this by default, but I'll check into it with #372.